src/jailer/src/env.rs

```diff
@@ -320,6 +380,8 @@ impl Env {
         } else {
             None
         };
+        #[cfg(target_arch = "aarch64")]
+        self.copy_cache_info()?;
```

Comment:
Are we doing this by default ? It would make sense to have a switch for it.

Reply:
You mean check whether or not the cache info is already present?

Reply:
We synced offline on this. The problem here is that the cache information will be copied into the chroot irrespective of whether Firecracker needs them or not (for example, upon resuming a microVM from a snapshot we do not need this info but we still copy the cache information into the chroot). Unfortunately, there is no easy way to check from inside the jailer logic what is the purpose of the run. Making one available is not super obvious and we decided to log an issue for this and continue with the current functionality for the moment.

Reply:
If this would be conditional, the user would need to take extra care to only disable it when they are sure no other snapshots will be done. Not sure if the effort is worth it.

In the context of incremental snapshots it would look weird to have this extra knob that _disables further snapshots_ but buys you some microseconds.